src/Raven.Server/Config/Categories/SecurityConfiguration.cs

```diff
@@ -89,11 +89,16 @@ public class SecurityConfiguration : ConfigurationCategory
         [ConfigurationEntry("Security.UnsecuredAccessAllowed", ConfigurationEntryScope.ServerWideOnly)]
         public UnsecuredAccessAddressRange UnsecuredAccessAllowed { get; set; }
 
-        [Description("Allow to specify well known certificate thumbprints that will be trusted by the server as cluster admins.")]
+        [Description("Well known certificate thumbprints that will be trusted by the server as cluster admins.")]
         [DefaultValue(null)]
         [ConfigurationEntry("Security.WellKnownCertificates.Admin", ConfigurationEntryScope.ServerWideOnly)]
         public string[] WellKnownAdminCertificates { get; set; }
 
+        [Description("Well known issuer thumbprints that will be used to validate a new certificate's HPKP if the issuer's certificate has changed.")]
```

Comment:
```suggestion
        [Description("Well known issuer public key hashes that will be used to validate a new certificate's PKP if the issuer's certificate has changed.")]
```

Reply:
`openssl s_client -servername $1 -connect $1:443 | openssl x509 -pubkey -noout | openssl rsa -pubin -outform der | openssl dgst -sha256 -binary | openssl enc -base64`

Reply:
It's too much to put in the configuration descriptiopn. I'll document it instead.